docassemble/assemblylinewizard/interview_generator.py

```diff
@@ -199,17 +198,18 @@ def source(self)->str:
         """
         Render and return the source of all blocks in the interview as a YAML string.
         """
+        if getattr(self, 'template_path'):
+          self._load_templates(self.template_path)
+        else:
+          self._load_templates("docassemble.assemblylinewizard:data/sources/interview_structure.yml")
         text = ""
         for block in self.blocks + self.questions.elements:
           text += "---\n"
           imports = self.templates.get('mako template imports',[])
           local_imports = self.templates.get('mako template local imports',[])
           formatted_local_imports = [ mako_local_import_str(user_info().package, import_key, local_imports[import_key]) for import_key in local_imports ]
           imports = imports + formatted_local_imports
-          if imports:
-            text += block.source(self.templates.get(block.template_key), imports=imports)
-          else:
-            text += block.source(self.templates.get(block.template_key))
+          text += block.source(self.templates.get(block.template_key), imports=imports)
```

Comment:
This tiny refactor will just let us be more flexible about the source of templates in the future--possibly making it a little simpler to add a custom template for a single block.